Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.15 as builder
+WORKDIR /go/src/github.com/gojekfarm/stevedore
+COPY ./ ./
+RUN make compile
+
+FROM alpine:3.12.4
+ENV HELM_VERSION="v3.4.0"
+RUN wget -q https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz -O - | tar -xzO linux-amd64/helm > /usr/local/bin/helm
+RUN chmod +x /usr/local/bin/helm
+RUN helm repo add stable https://charts.helm.sh/stable
+RUN helm repo update
+
+COPY --from=builder /go/src/github.com/gojekfarm/stevedore/out/stevedore /usr/local/bin/
+RUN addgroup -S appgroup && adduser -S stevedore -G appgroup
```

Comment:
We could use "stevedore" as group name as well.

Sometimes people use same UID and GID across all docker images throughout the organisation. We could make UID and GID some constant and make it configurable via build arg. So people can build image with their own GID and UID. This will also provide them a way to overcome UID and GID conflicts within the organisation easily.

Example: https://github.com/nginxinc/docker-nginx-unprivileged/blob/main/Dockerfile-debian.template

Reply:
Makes sense @arunvelsriram ... I will make the necessary changes and update the PR

Reply:
Done... Pls review it @arunvelsriram